tygron-environment/src/GOAL/GoalTestTygron.mas2g

```diff
@@ -4,8 +4,8 @@ environment{
 		%stakeholder is mandatory, as is the map, if slot is not set it will result in a random slot.
 		% Stakeholder, 0 is muncipianity, 1 is the housing corporation, 2 is company
 		stakeholder = 2,
-		map = "testmap",
```

Comment:
Pl0x this is not good :sob: Please change to "example" and the slot default to "-1" and lets leave it at that.